xp/xconnect/SIFconfig.json

```diff
@@ -0,0 +1,648 @@
+// -------------------------------------------------------------------------- //
```

Comment:
wat is dit nu voor file?
We zouden de originele installatie files moeten gebruiken, en evt alleen een SIF JSON file toevoegen die een delta toepast.

Reply:
Was niet op de hoogte dat je met SIF ook met delta's kan werken. Dat is dan inderdaad handiger.

Reply:
Aparte SIF file bedoel ik met "delta".